app/views/katello/api/v2/packages/show.json.rabl

```diff
@@ -1,11 +1,4 @@
 object @resource
 
 extends "katello/api/v2/packages/base"
-
-attributes :description
-attributes :license, :buildhost, :vendor, :relativepath
-attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group, :requires, :provides, :files
-
-node :human_readable_size do |package|
-  number_to_human_size(package.size)
-end
+extends "katello/api/v2/packages/backend", :object => Katello::Pulp::Rpm.new(@resource.uuid)
```

Comment:
I think its cleaner the way i did it, but i don't really have any strong argument either way.